### PR TITLE
Paintroid-452 Flutter: add languages + Crowdin Integration

### PR DESCRIPTION
**Pull request by ikrsk**
Ticket: [https://jira.catrob.at/browse/PAINTROID-452](https://jira.catrob.at/browse/PAINTROID-452)

### CODE REVIEW

File: flutter_launcher_icons.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 flutter_icons:
-  image_path: "assets/icon/app_icon.png"
+  image_path: "asset/icon/app_icon.png"
   android: true
   ios: true
-  adaptive_icon_foreground: "assets/icon/app_icon_foreground.png"
+  adaptive_icon_foreground: "asset/icon/app_icon_foreground.png"
```
Comment (juliajulie95) [inline flutter_launcher_icons.yaml:5]: Why the renaming?

Comment (ikrsk) [reply flutter_launcher_icons.yaml:5]: To maintain consistency across the project. All the other directories are named in the singular form.

Comment (juliajulie95) [reply flutter_launcher_icons.yaml:5]: I think flutter always uses 'assets', so I would stay with the flutter convention

**Comment by ThorstenBandel** [comment]
Fixed by https://github.com/Catrobat/Paintroid-Flutter/pull/13

**Pull request closed without merging**